kroxylicious-filters/kroxylicious-encryption/pom.xml

```diff
@@ -42,6 +42,13 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.ben-manes.caffeine</groupId>
+            <artifactId>caffeine</artifactId>
+            <version>${caffeine.version}</version>
+            <!-- provided by Kafka >= 3.6.0-->
+            <scope>provided</scope>
```

Comment:
But we don't run in the broker so we wont have it on the runtime classpath unless manually added. I think its a compile scope dependency for us.

Reply:
D'oh